### DISCUSSION TITLE
Monitoring with Prometheus podmonitor not working.

### DISCUSSION THREAD

**Discussion by ajrpayne** [fluxcd/flux2#2117] [Q&A]
I followed the "Monitoring with Prometheus" guide. But the metrics from all flux controls show target label "dropped" in the Prometheus UI. It shows up in service discovery as podMonitor/flux-system/flux-system/0 (0 / 11 active targets).

I can curl the /metrics endpoint from the containers.

**Comment by kingdonb** [comment]
Thanks for the report! See also: #2194 that fixed this, pinning the version before kube-prometheus-stack chart that broke, and prometheus-operator/prometheus-operator#4435 where the fix for this landed upstream.

From now on the guide should be working. Sorry for the issue that you experienced!
